app/_locales/en/messages.json

```diff
@@ -195,6 +195,21 @@
   "blockiesIdenticon": {
     "message": "Use Blockies Identicon"
   },
+  "nonceField": {
+    "message": "Show Nonce Field on Confirmation Screens."
+  },
+  "nonceFieldPlaceholder": {
+    "message": "Automatically calculate"
+  },
+  "nonceFieldHeading": {
+    "message": "Custom Nonce"
+  },
+  "nonceFieldDescription": {
+    "message": "Turn this on to change the nonce (transaction number) on confirmation screens. This is an advanced feature, use cautiously."
+  },
+  "borrowDharma": {
```

Comment:
It looks like this message was brought back accidentally in the merge conflict resolution - I deleted it recently.